pubspec.lock

```diff
@@ -73,7 +73,14 @@ packages:
       name: matcher
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.12.10"
+    version: "0.12.11"
+  material_color_utilities:
+    dependency: transitive
+    description:
+      name: material_color_utilities
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "0.1.3"
```

Comment:
Same here. changes in the `pubspec.lock` but there are no changes in the `pubspec.yaml`